src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1268,7 +1268,7 @@ def list_av_sets(cmd, resource_group_name=None):
     op_group = _compute_client_factory(cmd.cli_ctx).availability_sets
     if resource_group_name:
         return op_group.list(resource_group_name)
```

Comment:
What will be returned if list by resource group ?

Reply:
Contains VM list.